diff -urNp -x '*.orig' importlib_metadata-1.4.0.org/importlib_metadata/tests/fixtures.py importlib_metadata-1.4.0/importlib_metadata/tests/fixtures.py
--- importlib_metadata-1.4.0.org/importlib_metadata/tests/fixtures.py	2020-01-11 01:18:29.000000000 +0100
+++ importlib_metadata-1.4.0/importlib_metadata/tests/fixtures.py	2021-02-28 13:41:54.720031137 +0100
@@ -12,9 +12,9 @@ try:
 except ImportError:
     from contextlib2 import ExitStack
 
-try:
+if sys.version_info > (3, 5): # must be compatible with importlib_metadata._compat
     import pathlib
-except ImportError:
+else:
     import pathlib2 as pathlib
 
 
